src/codeu/chat/client/ClientUser.java

```diff
@@ -43,20 +44,25 @@ public ClientUser(Controller controller, View view) {
     this.controller = controller;
     this.view = view;
   }
-
-  // Validate the username string
-  static public boolean isValidName(String userName) {
-    boolean clean = true;
-    if (userName.length() == 0) {
+    
+// Validate the username string
+public boolean isValidName(String userName) {
+	boolean clean = true; 
```

Comment:
nit about whitespace. According to the style guide, all lines in the same block should have the same indentation. So this should be:
public boolean isVa....
  boolean clean...
  if (userName... 
etc